logstash-core/lib/logstash/util/wrapped_acked_queue.rb

```diff
@@ -291,10 +294,10 @@ def each(&blk)
         # TODO: for https://github.com/elastic/logstash/issues/6055 = will have to properly refactor
         @iterating = true
         @originals.each do |e, _|
-          blk.call(e) unless e.cancelled?
+          blk.call(e) unless (e.respond_to?(:cancelled?) && e.cancelled?)
```

Comment:
ah, now I know what it may have been. Shutdown signal events. I'll clean this up